src/style-spec/expression/evaluation_context.js

```diff
@@ -34,27 +34,27 @@ class EvaluationContext {
         this.featureDistanceData = null;
     }
 
-    id() {
+    id(): null | any {
```

Comment:
Any way to avoid `any` here? In theory this should be `number | null`.

Reply:
Narrowing feature id down to `number | null` also leads to the incompatibility with the style-spec expression `Value`